mongo/mongo.go

```diff
@@ -84,7 +84,7 @@ func (m *Mongo) GetBooks() (models.Books, error) {
 
 	books := &models.Books{}
 	if err := list.All(&books.Items); err != nil {
-		log.Event(nil, "can't get it", log.FATAL, log.Error(err))
+		log.Event(nil, "unable to retrieve books", log.FATAL, log.Error(err))
```

Comment:
we should be passing context into mongo method to be used by our logs.

Also i think this should be an error log not fatal.

Reply:
Good point, thank you 🙂 
Is this one of those where you'd use `ctx := context.Background()` or should I be passing the `ctx`?